backend/app/AppComponents.scala

```diff
@@ -57,20 +59,29 @@ class AppComponents(context: Context, config: Config)
   // TODO MRB: should we have allowed hosts enabled? how could it point to the ELB?
   val disabledFilters: Set[EssentialFilter] = Set(allowedHostsFilter)
 
+  // Play includes an akka actorSystem but due to licensing constraints we can only use it for play specific tasks
+  // so here we create a pekko actor system
+  private val pekkoActorSystem: PekkoActorSystem = PekkoActorSystem.create("pfi")
+  applicationLifecycle.addStopHook(() => {
+    logger.info("Shutting down pekko")
+    pekkoActorSystem.terminate()
+  })
+
   override def httpFilters: Seq[EssentialFilter] = {
     super.httpFilters.filterNot(disabledFilters.contains) ++ Seq(
       new AllowFrameFilter,
+      //Note: still using akka (instead of pekko) materializer from Play as both filters extend Play's Filter, so the types need to match
       new RequestLoggingFilter(materializer),
```

Comment:
could we remove the materializer entirely? does play have a default?

Reply:
It seems that DevEx has also kept the the materializer for their HstsFilter class